### PR TITLE
Add PSL Rifle

### PR DESCRIPTION
**Pull request by DPavonis**
#### Summary
Content "Added PSL rifle"

#### Purpose of change

Adds in a Romanian PSL as a somewhat rare option for players who want a semi-automatic rifle in 7.62x54R with detachable box magazines. The reason I chose the PSL specifically is that it seems to have been imported into the US in significant enough numbers under various names that it wouldn't be too far-fetched to see one show up in a gun store.

#### Describe alternatives you've considered

~~Somehow make a case for a Dragunov PR despite the fact actual Dragunovs are pretty rare in the US.~~

#### Testing

Fire up game, make sure the rifle and its magazines show up in debug, ~~pray I understand how item distribution works~~

#### Additional context

~~I'm not 100% confident on the volumes and weights of the magazines, these are just my best guesses based off taking measurements off of images for the volume and guesstimating for the weight.~~

Special thanks to Tonkatsu for pointing me toward #37374 and giving me a better method for determining gun rarity.

### CODE REVIEW

File: data/json/items/gun/762R.json [hunk 1]
```diff
@@ -114,5 +114,43 @@
...
+    "valid_mod_locations": [
+      [ "accessories", 4 ],
+      [ "barrel", 1 ],
+      [ "bore", 1 ],
+      [ "brass catcher", 1 ],
+      [ "grip", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights mount", 1 ],
```
Comment (TheShadowFerret) [inline data/json/items/gun/762R.json:148]:
```suggestion
      [ "sights mount", 1 ],
```
I don't think a DMR is going to need you to add a mount to it for a scope(esp. as Century includes an optic with their imports of it https://www.centuryarms.com/psl-54c-semi-auto-rifle-cal-7-62x54r.html) 🤔

Comment (DPavonis) [reply data/json/items/gun/762R.json]: Fair enough. I was thinking of just a bare, unmodified PSL, but eh, works for me. ¯\_(ツ)_/¯

**Comment by tenmillimaster** [comment]
Could you include a default mod of a sights adapter, as opposed to natively having a sights slot on the gun?

**Comment by Night-Pryanik** [comment]
@DPavonis could you please resolve conflicts?

File: data/json/items/gun/762R.json [hunk 2]
```diff
@@ -112,5 +112,44 @@
...
+  },
+  {
+    "id": "psl",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "//": "Mom can we have a McDragunov? Mom: we have McDragunov at home. The McDragunov at home:",
+    "type": "GUN",
+    "name": { "str": "PSL" },
+    "description": "The Romanian PSL is a designated marksman's rifle developed by Romania after political tensions with Soviet Russia forced them to develop their own alternative to the SVD.",
+    "weight": "4100 g",
```
Comment (nopenoperson) [inline data/json/items/gun/762R.json:124]: weight without a pso1 style scope should be about 3600g

**Review by nopenoperson — CHANGES_REQUESTED** [review]

**Pull request merged**